src/harness/fourslash.ts

```diff
@@ -831,6 +831,7 @@ module FourSlash {
                 if (expectedText !== undefined) {
                     assert.notEqual(actualQuickInfoText, expectedText, this.messageAtLastKnownMarker("quick info text"));
                 }
+                // TODO: should be '==='?

```

Comment:
i would switch them to just `if( expectedDocumentation)` and please remove comments